config/cesm/config_archive.xml

```diff
@@ -92,40 +92,32 @@
   </comp_archive_spec>
 
   <comp_archive_spec compname="cism" compclass="glc">
-    <rest_file_extension>\.[ri]\..*</rest_file_extension>
-    <hist_file_extension>\.h\..*\.nc$</hist_file_extension>
-    <hist_file_extension>\.initial_hist\..*\.nc$</hist_file_extension>
-    <rest_history_varname>unset</rest_history_varname>
-    <rpointer>
+    <rest_file_extension>[ri]</rest_file_extension>
+    <hist_file_extension>h\d*</hist_file_extension>
+    <hist_file_extension>initial_hist</hist_file_extension>
+      <rest_history_varname>unset</rest_history_varname>
+      <rpointer>
```

Comment:
Are these spaces correct? I think that `<rpointer_file>` and `<rpointer_content>` should be nested under `<rpointer>` but `<rpointer>` and `<rest_history_varname>` are at the same level as `<hist_file_extension>`